modules/cluster-logging-rn-5.5.1.adoc

```diff
@@ -0,0 +1,17 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-1"]
+= Logging 5.5.1
```

Comment:
The intent is to release https://issues.redhat.com/browse/LOG-2574 as part of 5.5.1.  Do we need to document that here some place?

Reply:
No, with the release of 5.5.1 being pushed out, it's release notes will get their own PR. This file is commented out for use later.